src/components/Timeline/Publication/Actions/Collect/CollectModule.tsx

```diff
@@ -0,0 +1,422 @@
+import { LensHubProxy } from '@abis/LensHubProxy';
```

Comment:
**question:** can we reuse this from `Publication` folder?

Reply:
Done